server/e2e_test.go

```diff
@@ -0,0 +1,572 @@
+package server
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cashshuffle/cashshuffle/message"
+
+	"github.com/avast/retry-go"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	basicPoolSize = 3
+	someAmount    = uint64(100000000)
```

Comment:
NIT: maybe testAmount and testVersion?

Reply:
good one. I just like to show that the values are arbitrary but I forgot these will be polluting namespace also.

ca52bf0